### ISSUE TITLE
Lenker fra forskjellige steder i registrering blir lagt i samme boks på detaljsiden

### ISSUE THREAD

**Issue by shenluna** [Informasjonsforvaltning/dataset-catalog#74]
@shenluna commented on [Wed May 12 2021](https://github.com/Informasjonsforvaltning/fdk-portal/issues/905)

@shenluna commented on [Wed May 12 2021](https://github.com/Informasjonsforvaltning/dataset-catalog-gui/issues/155)

Gjelder https://www.staging.fellesdatakatalog.digdir.no/datasets/04edc67b-046c-37a8-9822-29f03d2f1e80

![image.png](https://images.zenhubusercontent.com/5f9a823ccff8726db3db553b/6d0f1b14-66d8-40f5-9385-2bd742e35bef)

---

@azaxarov commented on [Wed May 12 2021](https://github.com/Informasjonsforvaltning/dataset-catalog-gui/issues/155#issuecomment-839801014)

Dette er en bug i fdk-portal

**Comment by azaxarov** [comment]
Er det riktig at informasjonsmodeller skal under `dct:conformTo` feltet? @stigbd @NilsOveTen 

Se følgende linje for mer info: https://github.com/Informasjonsforvaltning/dataset-catalog/blob/3afce9926b7295fc85c06a25275b33098672d60a/src/main/kotlin/no/fdk/dataset_catalog/rdf/RDFDatasetUtils.kt#L52

**Comment by stigbd** [comment]
Ja. Sjå https://data.norge.no/specification/modelldcat-ap-no/#klasse-standard

**Comment by shenluna** [comment]
Lukker denne saken da det er riktig i henhold til spesifikasjonen
